terraform/lotus-mainnet/main.tf

```diff
@@ -0,0 +1,35 @@
+terraform {
+  required_version = ">= 1.2"
+
+  backend "s3" {
+    bucket                      = "forest-iac"
+    key                         = "lotus-mainner/terraform.tfstate"
+    region                      = "us-west-1"
+    endpoint                    = "fra1.digitaloceanspaces.com"
+    skip_credentials_validation = true
+    skip_metadata_api_check     = true
+  }
+}
+
+module "lotus-calibnet" {
+  source = "../modules/forest_node"
```

Comment:
I still need clarification. If the module is not specific to Forest, let's rename it. Perhaps `filecoin_node`.